report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+
+
+
+```{r data, include=FALSE}
+beverage_sales <- read.csv("../rawdata/june1data.csv") %>% 
+   select(Count, DofW, Site, Intervention, ZeroCal, Sugary, Total)
+# head(beverage_sales)
+# dim(beverage_sales)
+```
+
+```{r Summarize Data, eval=FALSE, include=FALSE}
+# summarize_data(beverage_sales)$numerical
+# unique(beverage_sales$Site)
+# unique(beverage_sales$Intervention)
+```
+
+## 3. Exploratory Analysis
+
+Exploratory Data Analysis (EDA) is an essential precursor to formal statistical methods, uncovering underlying patterns and outliers in the data. It ensures the subsequent analysis is built on a solid foundation of understanding, thereby enhancing the reliability of the findings.
+
+
+### 3.1 Box Plots
+
+The following boxplots represent the distribution of sugary and zero-calorie beverage sales across different intervention strategies. Each boxplot captures the sales data variability with the central line denoting the median, the edges of the box indicating the interquartile range (IQR), and the whiskers extending to the furthest points that are not considered outliers. Outliers are individual points beyond the whiskers. Note that due to the bimodality of sales, the boxplot incorrectly indicates many outliers. The interventions, labelled on the x-axis, include baseline (preint), discount & messaging (dismes), washout period (wash), the discount only (dis), second washout (wash2), follow up (follow), calorie content poster (cal), the exercise required to burn the calorie content (excer), and combination of the two (both). 
+
+```{r Boxplots, fig.cap="The boxplots display the sales distribution of Sugary and Zero-Calorie beverages across various interventions. Each intervention is colour-coded and shows the range of sales data with the central line representing the median sales.", echo=FALSE, fig.height=3, fig.width=6, fig.pos="H"}
+boxplots_by_category(beverage_sales, responses = c("ZeroCal", "Sugary"), category = "Intervention")
+```
+
+
+### 3.2 Correlation Plot 
+  
```

Comment:
Like what the following subsection started off with, I feel it is important to discuss why these plots are generated here first instead of directly going into "here is a plot I did and let's interpret what it does". The "why" might be more interesting to the client.